### DISCUSSION TITLE
Security disclosure via email

### DISCUSSION THREAD

**Discussion by meyfa** [renovatebot/renovate#27397] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

On the 2024-02-10 at approx. 20:15 UTC, I've sent two emails regarding a potential security issue to the email address provided in SECURITY.md. Since then, there has been no response (or it didn't arrive in my inbox). I would like to inquire whether the email address is up-to-date and hope that the reports can be handled in a timely manner. Thanks!

**Comment by rarkins** [comment]
I'm a recipient of that address but haven't received anything. Please email the same address one more time but also include my personal email address at the same time (see my github profile)

**Reply by meyfa** [reply]
I have created a report under the GitHub security advisories, as suggested by @viceice.

**Comment by viceice** [comment, marked as answer]
you can also use this form to report

https://github.com/renovatebot/renovate/security/advisories

**Reply by meyfa** [reply]
Thank you, I have created a report there.
